uniffi_bindgen/src/scaffolding/templates/ForwardUniFFIScaffolding.rs

```diff
@@ -0,0 +1,19 @@
+// Create a macro that can be used by a dependent create to forward all of the scaffolding
```

Comment:
We might as well be extra redundant and link to the rust-lang issue here too, so the reason for this file isn't a mystery unless you have the additional context.